daemon/portmanager.cpp

```diff
@@ -1229,6 +1232,7 @@ int32_t PortManager::SetPortProperty(
 	    return EBUSY;
 	}
     }
+#ifndef ANDROID
```

Comment:
Please avoid a #ifndef ANDROID in the port manager. Commit [c999be7](https://github.com/intel/hdcp/commit/c999be7eda5a9a7f1dd12a9c92c0fbad324f8b14) made the PortManager class inheritable. Anything specific to one compositor should be moved to the PortManagerHWComposer or PortManager classes.